go.mod

```diff
@@ -1,26 +1,28 @@
 module github.com/Magic-Pod/magicpod-api-client
 
-go 1.17
+go 1.21
 
 require (
 	github.com/go-resty/resty v0.0.0-00010101000000-000000000000
-	github.com/mholt/archiver v3.1.1+incompatible
+	github.com/mholt/archiver/v3 v3.3.2
```

Comment:
cf: https://christina04.hatenablog.com/entry/go-modules-major-version (sorry it's a Japanese article)

It caused a large differences in `go.sum` and increased the binary file size, but it seems inevitable.